test/public/scripts/compiled/main.js

```diff
@@ -1,11 +1,6 @@
 console.log('dep1');
-
 console.log('Dependency 2 loaded');
-
 console.log('Dependency 3 loaded');
-
-console.log('Dependency 4 loaded');
-
-//= require dep1.js
+console.log('Dependency 4 loaded'); //= require dep1.js
```

Comment:
And this is just a lack of newlines in the replaced code. If we don't have control of that, I guess it's fine.